infra/Pos/Slotting/Types.hs

```diff
@@ -23,14 +33,65 @@ data EpochSlottingData = EpochSlottingData
 instance NFData EpochSlottingData
 
 -- | Data necessary for slotting to work which is basically part of GState.
-data SlottingData = SlottingData
-    { sdPenult      :: !EpochSlottingData
-    -- ^ 'EpochSlottingData' for penult (i. e. 'sdPenultEpoch') epoch.
-    , sdLast        :: !EpochSlottingData
-    -- ^ 'EpochSlottingData' for last epoch.
-    -- Last means 'sdPenultEpoch + 1'.
-    , sdPenultEpoch :: !EpochIndex
-    -- ^ Penult epoch for which 'EpochSlottingData' is known.
-    } deriving (Eq, Show, Generic)
+-- TODO: Reflect the fact that this will always have 2+ elements
+-- TODO: Make this a newtype wrapper
```

Comment:
By the way, I'm not sure if this datatype has serialization tests, but when you turn a `newtype/data` into a `type`, remove it from any `BinarySpec/SafeCopySpec` files you can find as it's not worth testing type synonyms for `HashMap`s, for example.